docs/core-concepts/api-routes.md

```diff
@@ -162,95 +158,123 @@ SolidStart makes it easy to implement a GraphQL API. The `graphql` function take
 
 ```tsx twoslash filename="routes/api/graphql.ts"
 import { APIEvent } from "solid-start/api";
-const graphql = (schema: string, resolvers: any) => 
-  (event: APIEvent) => { return new Response("GraphQL Response") }
+const graphql = (schema: string, resolvers: any) => (event: APIEvent) => {
+  return new Response("GraphQL Response");
+};
 
 const schema = `
   type Query {
     hello: String
   }
 `;
 
-const handler =  graphql(schema, {
+const handler = graphql(schema, {
   Query: {
     hello: () => "Hello World"
   }
 });
 
-export const get = handler;
+export const GET = handler;
+
+export const POST = handler;
+```
+
+## Using tRPC In Solid
 
-export const post = handler;
+First, install the required [tRPC] packages & Zod
 
+```bash
+npm install @trpc/server@next @trpc/client@next zod
 ```
 
+Then, install the required Solid packages
 
-## Exposing a tRPC Server route
+```bash
+npm install @tanstack/solid-query solid-trpc@next solid-start-trpc@latest
+```
```

Comment:
I would like the trpc basic example to show how to integrate the API routes with trpc directly, coz the point is to show how to use API routes. We might have a more extensive guide for integrations, but this doc is for the API routes functionality. Want to show how to do the raw integration if people want to do that. We can make a note about these libraries towards the end